Jenkinsfile

```diff
@@ -1,168 +0,0 @@
-#!groovy
```

Comment:
I am sure you don't want to delete this just yet.

Reply:
hah nope, `git checkout master Jenkinsfile` will fix it